### PR TITLE
chore: added npmrc save-exact true and set code ownership tslint and tsconfig

### PR DESCRIPTION
**Pull request by valorkin**
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Comment by valorkin** [comment]
> looks good. I also used to remove these ^, ~ as it can be dangerous

removed

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# This is a comment.
```
Comment (droshev) [inline .github/CODEOWNERS:1]: why do we need this?

**Comment by valorkin** [comment]
> Please remove the codeowners file

I can replace this with additional rule to mergeable, but changes to tslint and tsconfig has to under strict control
https://github.com/mergeability/mergeable#dependent

What do you think?
cc @mikerodonnell89 @droshev

**Comment by InnaAtanasova** [comment]
@valorkin the PR needs a rebase
@mikerodonnell89 can you take a look at it? Your approval is required for the PR to be merged.

**Pull request merged**